schemas/telemetry/crash/crash.4.schema.json

```diff
@@ -1470,18 +1470,33 @@
             "threads": {
               "items": {
                 "properties": {
-                  "ip": {
-                    "pattern": "^(0x[a-fA-F0-9]+|0)$",
-                    "type": "string"
-                  },
-                  "module_index": {
-                    "minimum": 0,
-                    "type": "integer"
-                  },
-                  "trust": {
-                    "type": "string"
+                  "frames": {
+                    "items": {
+                      "properties": {
+                        "ip": {
+                          "pattern": "^(0x[a-fA-F0-9]+|0)$",
+                          "type": "string"
+                        },
+                        "module_index": {
+                          "minimum": 0,
+                          "type": "integer"
+                        },
+                        "trust": {
+                          "type": "string"
+                        }
+                      },
+                      "required": [
+                        "ip",
+                        "trust"
+                      ],
```

Comment:
I'm not sure if this is the right thing-- it looks like it has no change on things as they stand (all crash pings with stack traces have these values, contained inside a frames object) but may artificially restrict us from modifying these pings in the future.

Reply:
let's not do this after all